### DISCUSSION TITLE
What if Rails was Built on Python?

### DISCUSSION THREAD

**Discussion by Allin2000** [loco-rs/loco#548] [General]
I like loco.

**Comment by Allin2000** [comment]
I like fastapi, but I think it needs loco.

**Comment by Allin2000** [comment]
Axum has 16,000 stars. Fastapi has 70,000 stars.This means that if there is a Loco written in Python, it will have 15,000 stars.

**Comment by Allin2000** [comment]
After my careful research on loco. Rust is not suitable for side projects and startups. I suggest you consider fastapi. Rapid development is more important than performance.

**Comment by jondot** [comment]
Thanks, i will close this for now.
